### PR TITLE
Added Dev1 text

### PR DESCRIPTION
**Pull request by mihaicostin34**
### Pull Request Overview

This pull request adds Dev1

### Testing Strategy

This pull request was tested by running `npm run start`

### TODO or Help Wanted

This pull request still needs a link for the homework assignment and maybe more detailing on compiling and execution

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by: Mihai Costin <mihaicostin342@gmail.com>

### CODE REVIEW

File: docs/devoir/01.md [hunk 1]
```diff
@@ -1,5 +1,439 @@
...
+    $./gradlew build
+```
+
+Pour exécuter le programme:
+```
+    $./gradlew run --args="<nom_du_fichier_d'entrée>"
+```
+
+## Règles
+1. Vous ne pouvez pas utiliser RegEx ou toute autre bibliothèque qui utilise des expressions régulières (sauf \n\r? pour séparer les lignes d'entrée de la source).
```
Comment (alexandruradovici) [inline docs/devoir/01.md:425]:
```suggestion
1. Vous ne pouvez pas utiliser `RegEx` ou toute autre bibliothèque qui utilise des expressions régulières (sauf \n\r? pour séparer les lignes d'entrée de la source).
```

File: docs/devoir/01.md [hunk 2]
```diff
@@ -1,5 +1,439 @@
...
+2. Le code source doit être indenté (-0.1p).
+3. Vous devez expliquer en détail dans le Readme.md comment vous avez rédigé les devoirs (-1p)
+
+## Tricher
+Les devoirs sont individuels. Toute tentative de tricher entraînera 0p pour les devoirs. Un système anti-copie automatisé sera utilisé.
```
Comment (alexandruradovici) [inline docs/devoir/01.md:430]: use `:::error`

**Comment by mihaicostin34** [comment]
> Looks really good, please add a full example, the input file and the output text.

Added a basic example

File: docs/devoir/01.md [hunk 3]
```diff
@@ -1,5 +1,463 @@
...
+Pour exécuter le programme:
+```
+    $./gradlew run --args="<nom_du_fichier_d'entrée>"
+```
+
+## Exemple
+Pour le fichier `.asm` suivant:
+
+```nasm showLineNumbers
+    push 20
```
Comment (alexandruradovici) [inline docs/devoir/01.md:428]: Please add a comment explaining what each instruction does.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/devoir/01.md [hunk 4]
```diff
@@ -1,5 +1,469 @@
...
+
+:::warning
+    Deadline: le 17 mars 
+
+    Points: 2 points de la note finale
+
+    Lien: https://classroom.github.com/a/s3tmWRAY
+
+    Téléchargement tardif: **1p/jour** (maximum 2 jours) 
+:::
```
Comment (alexandruradovici) [inline docs/devoir/01.md:18]:
```suggestion
:::info
    Deadline: le 17 mars \
    Points: 2 points de la note finale \
    Lien: https://classroom.github.com/a/s3tmWRAY \
    Téléchargement tardif: **1p/jour** (maximum 2 jours) 
:::
```

File: docs/devoir/01.md [hunk 5]
```diff
@@ -1,5 +1,469 @@
...
+        <td>Instruction</td><td>Paramètres</td><td>Description</td><td>Exemple</td>
+    </tr>
+    </thead>
+    <tr>
+        <td>`push`</td>
+        <td>numéro entier </td>
+        <td>Elle ajoute le paramètre sur la pile </td>
+        <td>
+            ```nasm
+               ; la pile est [ ]
```
Comment (alexandruradovici) [inline docs/devoir/01.md:58]: Comments are used with `;`, but in the examples are with `#`. I suggest using one of them.

Please add a statement explaining that every line that starts with the comment symbol should be ignored.

File: docs/devoir/01.md [hunk 6]
```diff
@@ -1,5 +1,469 @@
...
+Pour exécuter le programme:
+```
+    $./gradlew run --args="<nom_du_fichier_d'entrée>"
+```
+
+## Exemple
+Pour le fichier `.asm` suivant:
+
+```nasm showLineNumbers
+    push 20     #ajoute 20 à la pile
```
Comment (alexandruradovici) [inline docs/devoir/01.md:428]: I suggest placing all the comments into one single line and adding that comments will not be placed like that in the test files.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/devoir/01.md [hunk 7]
```diff
@@ -1,5 +1,469 @@
...
+1. Vous ne pouvez pas utiliser *RegEx* ou toute autre bibliothèque qui utilise des expressions régulières (sauf \n\r? pour séparer les lignes d'entrée de la source).
+2. Le code source doit être indenté (-0.1p).
+3. Vous devez expliquer en détail dans le Readme.md comment vous avez rédigé les devoirs (-1p)
+
+:::danger[Tricher]
+Les devoirs sont individuels. Toute tentative de tricher entraînera 0p pour les devoirs. Un système anti-copie automatisé sera utilisé. 
+:::
+
+## Questions
+Si vous avez des questions concernant les devoirs, posez-les en publiant un problème sur le [github repository pour les questions](https://github.com/UPB-FILS-ALF/questions) avec le format du titre [simulator]\<titre de votre question\>. Vous aurez besoin d'un compte github pour cela.
```
Comment (alexandruradovici) [inline docs/devoir/01.md:463]:
```suggestion
Si vous avez des questions concernant les devoirs, posez-les en publiant un problème sur le [github repository pour les questions](https://github.com/UPB-FILS-ALF/questions) avec le format du titre `[simulator]\<titre de votre question\>`. Vous aurez besoin d'un compte github pour cela. 
```

File: docs/devoir/01.md [hunk 8]
```diff
@@ -1,5 +1,469 @@
...
+                jump jumper ; ERROR (jump): undefined label jumper
+            ```
+        </td>
+    </tr>
+    <tr>
+        <td>invalid jump address </td>
+        <td>le simulateur essaie de faire un jump à une adresse (numéro de ligne) qui n'existe pas dans le fichier </td>
+        <td>
+            ```nasm
+                jump 2 ; ERROR (mul): invalid jump address 2
```
Comment (alexandruradovici) [inline docs/devoir/01.md:369]:
```suggestion
                jump 2 ; ERROR (jump): invalid jump address 2
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/devoir/01.md [hunk 9]
```diff
@@ -1,5 +1,467 @@
...
+## Information
+
+:::info
+    Deadline: le 17 mars 
+
+    Points: 2 points de la note finale
+
+    Lien: https://classroom.github.com/a/s3tmWRAY
+
+    Téléchargement tardif: **1p/jour** (maximum 2 jours)
```
Comment (alexandruradovici) [inline docs/devoir/01.md:17]:
```suggestion
    Deadline: le 17 mars \
    Points: 2 points de la note finale \
    Lien: https://classroom.github.com/a/s3tmWRAY \
    Téléchargement tardif: **1p/jour** (maximum 2 jours) 
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Pull request merged**